src/features/pages/home/live-pricing/components/live_market_table.tsx

```diff
@@ -0,0 +1,148 @@
+import Link from 'features/components/atoms/link'
+import { Localize } from 'components/localization'
+
+export type TLiveMarketTableProps = {
+    selected_market: TAvailableLiveMarkets
+    link_to: string
+    display_name: string
+}
+
+const LiveMarketTable = ({ selected_market, link_to, display_name }: TLiveMarketTableProps) => {
+    const { is_eu } = useRegion()
+    const [initial_loaded, setInitialLoaded] = useState(false)
+
+    const [rawMarketsData, setRawMarketsData] = useState()
+
+    const TABLE_VISIBLE_ROWS = 5
+    const [markets_data, setMarketsData] = useState(() => {
+        const temp = new Map<TAvailableLiveMarkets, TMarketData[]>()
+        return temp
+    })
+    const intervalRef = useRef(null)
+
+    const [sorting, setSorting] = React.useState<SortingState>([])
+
+    const region = useMemo(() => {
+        return is_eu
+            ? 'https://deriv-static-testing-default-rtdb.firebaseio.com/eu.json'
+            : 'https://deriv-static-testing-default-rtdb.firebaseio.com/row.json'
+    }, [is_eu])
+
+    const getData = useCallback(async () => {
+        const rawResponse = await fetch(region)
+        const response = await rawResponse.json()
+        setRawMarketsData(response)
+    }, [region])
+
+    useEffect(() => {
+        if (!initial_loaded) {
+            getData()
+            setInitialLoaded(true)
+        }
+        intervalRef.current = setInterval(getData, 10000)
+        return () => {
+            clearInterval(intervalRef.current)
+        }
+    }, [getData, initial_loaded])
+
+    const updateData = useCallback(() => {
+        if (rawMarketsData) {
+            const markets = rawMarketsData.market == 'indices' ? 'stocks' : rawMarketsData.market
+
+            Object.keys(markets).map((item) => {
+                if (item == selected_market) {
+                    const selected_market_data = rawMarketsData.market[item]
+                    const result = Object.values(selected_market_data)
+                    setMarketsData(result)
+                }
+            })
+        }
+    }, [rawMarketsData, selected_market])
+
+    useEffect(() => {
+        updateData()
+    }, [updateData])
+
+    const columns = useLiveColumns()
+
+    const table = useReactTable({
+        data: markets_data,
+        columns,
+        state: {
+            sorting,
+        },
+        getCoreRowModel: getCoreRowModel(),
+        getSortedRowModel: getSortedRowModel(),
+        onSortingChange: setSorting,
+    })
+
+    const rows = table.getRowModel().rows.slice(0, TABLE_VISIBLE_ROWS)
+
+    return (
+        <>
+            <Container.Fixed>
+                <Flex.Box justify="center" mt="16x">
+                    <table className="table-container">
+                        <thead>
+                            {table.getHeaderGroups().map((headerGroup) => (
+                                <tr key={headerGroup.id} className={table_row_header}>
```

Comment:
As far as I know we can use it like:
```suggestion
                                <FlexBox.Box as='tr' key={headerGroup.id} className={table_row_header}>
```
And pass most of css from scss file